src/Nethermind/Nethermind.Evm.Test/Tracing/GethLikeJavaScriptTracerTests.cs

```diff
@@ -44,7 +44,7 @@ public void log_operations()
                 GetTracer(userTracer),
                 MStore(),
                 MainnetSpecProvider.CancunActivation);
-        GethLikeTxTrace traces = tracer.BuildResult().First();
+        using GethLikeTxTrace traces = tracer.BuildResult().First();
```

Comment:
```suggestion
        GethLikeTxTrace traces = tracer.BuildResult().First();
```
Meant to be in this PR?